### DISCUSSION TITLE
Hide Jupyter Notebooks from cards

### DISCUSSION THREAD

**Discussion by VitorCMatias** [anuraghazra/github-readme-stats#1033] [Q&A] [answered]
Hi😉, 
I wish to hide **Jupyter notebooks** from my _Compact Language Card Layout_, but it is not working, what is the keyword for that "language"?

**Comment by anuraghazra** [comment, marked as answer]
You need to URL encode the string to be able to hide it `jupyter%20notebook`

We documented it too: https://github.com/anuraghazra/github-readme-stats#language-card-exclusive-options
![image](https://user-images.githubusercontent.com/35374649/117353621-f6f96400-aecd-11eb-80b6-9b34f214d8a1.png)

**Reply by VitorCMatias** [reply]
Thanks 😁
